src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/MockJupyterDAO.scala

```diff
@@ -5,8 +5,13 @@ import org.broadinstitute.dsde.workbench.model.google.GoogleProject
 import scala.concurrent.Future
 
 object MockJupyterDAO extends JupyterDAO {
-
   override def isProxyAvailable(googleProject: GoogleProject, clusterName: ClusterName): Future[Boolean] = Future.successful(true)
 
   override def isAllKernalsIdle(googleProject: GoogleProject, clusterName: ClusterName): Future[Boolean] = Future.successful(true)
 }
+
+class MockJupyterDAO(isUp: Boolean) extends JupyterDAO {
```

Comment:
It feels weird to have an object `MockJupyterDAO` and class `MockJupyterDAO` both extending `JupyterDAO`. Could we maybe just have the class with a default parameter for `isUp`?